packages/mgt-components/README.md

```diff
@@ -55,6 +55,161 @@ The components can be used on their own, but they are at their best when they ar
     <mgt-agenda group-by-day></mgt-agenda>
     ```
 
+
+## <a id="disambiguation">Disambiguation</a>
+
+MGT is built using [web components](https://developer.mozilla.org/en-US/docs/Web/Web_Components). Web components use their tag name as a unique key when registering within a browser. Any attempt to register a component using a previously registered tag name results in an error being thrown when calling `CustomElementRegistry.define()`. In scenarios where multiple custom applications can be loaded into a single page this created issues for MGT, most notably in developing solutions using SharePoint Framework.
+
+To mitigate this challenge we built the [`mgt-spfx`](https://github.com/microsoftgraph/microsoft-graph-toolkit/tree/main/packages/mgt-spfx) package. Using `mgt-spfx` developers could centralize the registration of MGT web components across all SPFx solutions deployed on the tenant. By reusing MGT components from a central location web parts from different solutions could be loaded into a single page without throwing errors. When using `mgt-spfx` all MGT based web parts in a SharePoint tenant used the same version of MGT.
+
+To allow developers to build web parts using the latest version of MGT and load them on pages along web parts that use v2.x of MGT, we've added a new disambiguation feature to MGT. Using this feature developers can specify a unique string to add to the tag name of all MGT web components in their application.
+
+### Usage in standard HTML and JavaScript
+
+The earlier example can be updated to use the disambiguation feature as follows:
+
+```html
+<script type="module">
+  import { Providers, customElementHelper } from '@microsoft/mgt-element';
+  customElementHelper.withDisambiguation('contoso');
+  import { Msal2Provider } from '@microsoft/mgt-msal2-provider';
+  Providers.globalProvider = new Msal2Provider({clientId: 'clientId'});
```

Comment:
```suggestion
  import { Providers, customElementHelper } from '@microsoft/mgt-element';
  import { Msal2Provider } from '@microsoft/mgt-msal2-provider';
  customElementHelper.withDisambiguation('contoso');
  Providers.globalProvider = new Msal2Provider({clientId: 'clientId'});
  
  // import the components using dynamic import to avoid hoisting
  import('@microsoft/mgt-components');
```